rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/misc.kt

```diff
@@ -155,16 +155,24 @@ fun literalToNumber(
     text: String,
     position: Position?
 ): Expression {
+    // fix minus at right
```

Comment:
This is not clear to me. What is the goal of this? Could you rephrase the comment please?

Reply:
This would be a nice example to add as a comment. Also, is it ok that "2-" is interpreted as the number 2 or should we produce an error?

Reply:
"2-" has to be interpreted as -2